pwndbg/color/__init__.py

```diff
@@ -73,3 +75,49 @@ def terminateWith(x, color):
 
 def ljust_colored(x, length, char=' '):
     return x + (length - len(strip(x))) * char
+
+
+try:
+    import pygments
+    import pygments.lexers
+    import pygments.formatters
+    from pwndbg.color.lexer import PwntoolsLexer
+except ImportError:
+    pygments = None
+
+pwndbg.config.Parameter('syntax-highlight', True, 'Source code / assembly syntax highlight')
+pwndbg.config.Parameter('syntax-highlight-style', 'monokai', 'Source code / assembly syntax highlight stylename of pygments module')
```

Comment:
We should probably create a `@pwndbg.config.Trigger` for this and check if user sets valid value - if not - set it to default and warn the user?

Reply:
As currently I can do `set syntax-highlight-style 'whatever'` and `context` will fail

Reply:
the style should actually be part of the 'theme' config instead of the global config as style for highlighting is exactly what theme is for :smile_cat:

Reply:
@anthraxx :+1: :heart:

Reply:
Added the trigger to restore value when the value is invalid. Also, use the `theme` config for the style setting.